cmd/itzo-launcher/itzo-launcher.go

```diff
@@ -99,16 +99,27 @@ func EnsureItzo() (string, error) {
 }
 
 func RunItzo(itzoPath string) error {
+	config, err := readCellConfig()
+	if err != nil {
+		// ignore
+	}
+	klog.Info(config)
 	klog.V(2).Infof("starting itzo")
 	logfile, err := os.OpenFile(
 		LogDir+"/itzo.log", os.O_RDWR|os.O_APPEND|os.O_CREATE, 0600)
 	if err != nil {
 		return fmt.Errorf("opening itzo logfile: %v", err)
 	}
 	defer logfile.Close()
+
+	cmdArgs := []string{"--v", "5"}
```

Comment:
If we want to use default arguments, we should do deduplication somewhere.

Reply:
added handling for defaults in `GetItzoFlags` :+1: